test/e2e-tests-auction.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+cd dex-contracts/
+
+# Place 6 orders in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
+truffle exec scripts/sell_order.js 1 2 3 12 12
+truffle exec scripts/sell_order.js 1 3 2 22 20
+truffle exec scripts/sell_order.js 1 1 3 15 10
+truffle exec scripts/sell_order.js 1 1 2 180 15
+truffle exec scripts/sell_order.js 1 2 1 4 52
+truffle exec scripts/sell_order.js 1 3 1 20 280
+
```

Comment:
Can you check here that the orders made it into MongoDB?

Reply:
Do you mean include some bash command that ensures the orders are in the database?

Reply:
Is it currently the case that deposit and withdraw tests are verifying this DB inclusion?

Reply:
Withdraws check the balance in the end. It would probably make sense to add intermediate checks for withdraw requests there as well. As mentioned, this will make finding the point of failure much simpler.

For this case I think it's particularly important, because we might accidentally break the listener functionality without noticing, since we don't have the full e2e functionality yet. By checking the database, we could make sure it's doing what it's supposed to do on every PR.

Reply:
Done!